controllers/nginx/rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -236,7 +236,19 @@ http {
         ssl_verify_client on;
         ssl_verify_depth {{ $location.CertificateAuth.ValidationDepth }};
         {{ end }}
+        
+        {{ if (or $location.Redirect.ForceSSLRedirect (and (not (empty $server.SSLCertificate)) $location.Redirect.SSLRedirect)) }}
```

Comment:
@jasonbrownbridge We all do :P I've removed all the passthrough stuff, as I use only HTTP/s traffic.

Anyway, I think the idea is to always redirect the SSL, as this is a global annotation, isn't it? 

This '/check', by the otherside could be inside the same Ingress Object (of the vhost that contains the TLS config). I suppose that Ingress Controller assumes the first created ingress and discards the conflicting one. (you have two ingress objects pointing to the same host, in this case www.example.com)

Also, by the previous template (before this commit), the HTTPs redirection would be inserted anyway in all existing 'path' rules :)

But let me know about your tests!

@aledbf Am I right with this conclusion?